src/extensions/g3d/model.ts

```diff
@@ -174,6 +174,23 @@ export const G3dSymbols = {
             const seqId = ctx.element.unit.model.coarseHierarchy.spheres.seq_id_begin.value(ctx.element.element);
             return info.haplotype[seqId] || '';
         }
+    ),
+    chromosome: QuerySymbolRuntime.Dynamic(CustomPropSymbol('g3d', 'chromosome', Type.Str),
```

Comment:
@dsehnal, looks like we don't expose the coarse properties, probably should

Reply:
Yeah, we should. Not in scope of this PR tho.